testDefinition/examplesTS/sleepChain.ts

```diff
@@ -1,7 +1,7 @@
 import * as suitest from '../../index';
 
 async function sleepTest() {
-	const sampleSleep = suitest.sleep(10);
+	const sampleSleep = suitest.sleep(null);
```

Comment:
sleep can accept null?

Reply:
on fe it is so, one can set null by `.` to sleep line